tests/php/test-fieldmanager-context-term.php

```diff
@@ -110,9 +122,13 @@ private function _get_html_for( $field, $test_data = null ) {
 	}
 
 	public function test_context_render_add_form() {
+		if ( ! _fm_phpunit_is_wp_at_least( 4.4 ) ) {
+			return $this->_skip_tests_because_version( 4.4 );
```

Comment:
There's another test class for FM term meta, where everything here is also tested using the old method. I'm skipping these tests as a convenience -- otherwise, I'd have to do a level of error-checking that just seems unnecessary, e.g. I couldn't call `$base->add_term_meta_box( 'test meta box', $this->taxonomy )->add_term_fields( $this->taxonomy );`, I'd have to:
```php
$context = $base->add_term_meta_box( 'test meta box', $this->taxonomy );
if ( $context ) {
    $context->add_term_fields( $this->taxonomy );
}
```
... then conditionally run the assertions.